prog/vm/nexus.rb

```diff
@@ -275,6 +274,11 @@ def before_run
     nap 1
   end
 
+  label def clean_prep
+    host.sshable.cmd("common/bin/daemonizer --clean prep_#{q_vm}")
```

Comment:
Just to be sure, we can clean multiple times, right? It's idempotent?

Reply:
good that you asked that. No, daemonizer was not designed that way best I can tell, patch forthcoming

Reply:
It's in the follow-up patch.